### ISSUE TITLE
[BUG] Music widget not showing spotify art

### ISSUE THREAD

**Issue by alvaro-munoz** [manilarome/the-glorious-dotfiles#101]
Apparently Spotify changed the URL for the art of the albums, but the updated URL does not show on the metadata info. In order to fix it you can add `| sed -e 's/open.spotify.com/i.scdn.co/g'` to the end of the `get_url_art` on the `spotify-music-updater.lua` file.
Don't know if many people use Spotify for it's music but I do and I think it's an easy fix.

**Comment by manilarome** [comment]
I labeled this as not a bug because the spotify-updater in the music widget will not really  work because it is not integrated in the music widget. I stopped integrating it because I don't know if I have to combine the mpd and spotify updater in the same widget or separate them. Anyway thanks for this `| sed -e 's/open.spotify.com/i.scdn.co/g'`. It will definitely help if I have to integrate the spotify-updater :) Closing this for now.
